compiler/noirc_evaluator/src/ssa/verus_vir_gen/expr_to_vir/exprs.rs

```diff
@@ -178,11 +183,11 @@ fn ssa_value_to_expr(
     let value_id = &dfg.resolve(*value_id);
     let value = &dfg[*value_id];
     match value {
-        Value::Instruction { instruction: _, position, typ } => {
-            param_to_expr(value_id, *position, typ, result_id_fixer, dfg)
+        Value::Instruction { instruction: _, position: _, typ } => {
+            param_to_expr(value_id, None, typ, result_id_fixer, dfg)
```

Comment:
Why?

Reply:
This fixes the illogical debug string which we had for values in the VIR structure. Now for a value 'v24' the debug string will be "(Span)v24" instead of "(Span)param position 0". We want to pass the position only if we are converting a function parameter into expr and not when we are converting a local variable like 'v24 = not v8'.